.github/workflows/codeql.yml

```diff
@@ -0,0 +1,41 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "main", "release/0.3", "release/0.4", "release/0.5", "release/0.6", "release/0.7", "release/0.8", "release/0.9", "release/0.10", "release/0.11", "release/0.12", "release/0.13", "release/0.14", "release/0.15" ]
```

Comment:
Is there some way to specify this with a wildcard? Seem odd that we'd have to manually list every release.

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-filters

Reply:
Looks like we should make the change to `release/**`, but in a follow-up PR.